integration-tests/tests/utils/config.py

```diff
@@ -0,0 +1,50 @@
+import shutil
+from dataclasses import dataclass
+from pathlib import Path
+
+
+@dataclass(frozen=True)
+class PackageConfig:
+    clp_bin_dir: Path
+    clp_package_dir: Path
+    clp_sbin_dir: Path
+    test_output_dir: Path
```

Comment:
how about test_root_dir, since it doesn't only contain output but also contains downloaded logs (i.e. input to the log)